docs/deploying-to-azure-containerapps.md

```diff
@@ -707,8 +749,7 @@ az containerapp create \
   --target-port 8087 \
   --min-replicas 1 \
   --env-vars NARRATION_AZURE_OPEN_AI_ENABLED=true \
-             NARRATION_AZURE_OPEN_AI_KEY="$AZURE_OPENAI_KEY" \
-             NARRATION_AZURE_OPEN_AI_ENDPOINT="$AZURE_OPENAI_ENDPOINT"
+             NARRATION_AZURE_OPEN_AI_KEY="$AZURE_OPENAI_KEY"
```

Comment:
`NARRATION_AZURE_OPEN_AI_*` properties aren't used anymore since we removed MS Semantic Kernel.

Deploying the narration service should be

```bash
az containerapp create \
      --resource-group "$RESOURCE_GROUP" \
      --tags system="$TAG_SYSTEM" application="$NARRATION_APP" \
      --image "${NARRATION_IMAGE}-azure-openai" \
      --name "$NARRATION_APP" \
      --environment "$CONTAINERAPPS_ENVIRONMENT" \
      --ingress external \
      --target-port 8087 \
      --min-replicas 1 \
      --env-vars QUARKUS_PROFILE=azure-openai \
                 QUARKUS_LANGCHAIN4J_AZURE_OPENAI_API_KEY="$AZURE_OPENAI_KEY" \
                 QUARKUS_LANGCHAIN4J_AZURE_OPENAI_RESOURCE_NAME="$COGNITIVE_SERVICE" \
                 QUARKUS_LANGCHAIN4J_AZURE_OPENAI_DEPLOYMENT_ID="$COGNITIVE_DEPLOYMENT"
```

Reply:
Fixed with https://github.com/quarkusio/quarkus-super-heroes/pull/570/commits/9a849aea6366bd4f8763884c6f6ca181e13bd2d2